query/branch/default_branch_blocks_force_push.sql

```diff
@@ -1,19 +1,19 @@
 select
   -- Required Columns
-  r.full_name as resource,
+  url as resource,
```

Comment:
What was the reasoning for changing the `resource` column value here, is the equivalent of `full_name` no longer available in the new table? Also, what are sample values for `r.full_name` and `url`?

Reply:
Honestly, I think it's because I'd seen a lot of `resource` cols in Sherlock and Compliance using `url` so thought it might be something that is clickable in one of the output templates - but then I didn't verify it... I do notice I didn't change all instances of `resource` across this mod either, so we can make a decision and I will update?

Reply:
I'm ok to switch to `url` if it's still unique across rows and we have `name_with_owner` as an additional dimension anyway